spec/features/users_spec.rb

```diff
@@ -1,21 +1,59 @@
 require 'rails_helper'
 include LogInHelper
 
-feature 'Users' do
-  before(:each) do
+feature 'Method 1: Instructor search a user' do
+  before(:all) do
```

Comment:
If #504 issue can be fixed, we can use ```before(:each)``` instead of ```before(:all)```. Now we cannot run ```student.save``` twice, because it will generate new users twice and occur an error describe in #504.